cmd/run.go

```diff
@@ -94,6 +95,7 @@ func bindRunFlags(cmd *cobra.Command, config *app.Config) {
 	cmd.Flags().StringVar(&config.ProcDirectory, "proc-directory", "", "Directory to look into in order to detect other stack components running on the host.")
 	cmd.Flags().StringVar(&config.ConsensusProtocol, "consensus-protocol", "", "Preferred consensus protocol name for the node. Selected automatically when not specified.")
 	cmd.Flags().StringVar(&config.Nickname, "nickname", "", "Human friendly peer nickname. Maximum 32 characters.")
+	cmd.Flags().StringVar(&config.BeaconNodeHeaders, "beacon-node-headers", "", "Comma separated list of headers formatted as header=value")
```

Comment:
I think it will be easier for you, if you follow the already existing structure for a comma separated list
```suggestion
	cmd.Flags().StringSliceVar(&config.BeaconNodeHeaders, "beacon-node-headers", "", "Comma separated list of headers formatted as header=value")
```

This will automatically map the value "x=val1,y=val2" to `[]string{"x=val1","y=val2"}`, instead of a string.

Of course, this will require some more changes in the code as now in your config you will have `[]string` for `BeaconNodeHeaders`, instead of `string`.

Reply:
Agree, great idea!